tests/plugins/clipboard/manual/pasteideographicspace.html

```diff
@@ -0,0 +1,12 @@
+
+<textarea style="height: 80px; width: 500px;">
+	全角スペースの削除が行われたり半角スペース変換が行われたりする
```

Comment:
Please remove tabs and new lines at the beginning and the end. Because of these characters, pasted text seems suspicious (it's full of `&nbsp;`) and makes testing harder.